packages/rancher-istio/charts/templates/istio-cni-psp.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: psp-istio-cni
+  namespace: istio-system
```

Comment:
Should this be `{{ template "istio.namespace" . }}`?

Reply:
This happens several times across the PR, so I'll avoid spam by just adding the comment here.

Reply:
Yes, this should be changed